sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/protobuf/ProtobufImporter.java

```diff
@@ -41,10 +42,15 @@
   @Override
   final void consume(T message) {
     InputFile inputFile = SensorContextUtils.toInputFile(context.fileSystem(), toFilePath.apply(message));
+
     // file may be null because it's not within the project base dir
     if (inputFile == null) {
       return;
     }
+
+    if (!filesProcessed.add(inputFile.path().toAbsolutePath())) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "path"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)